kod/object/item/passitem/questitem/srsttkn.kod

```diff
@@ -19,8 +19,10 @@ resources:
    StatsResetToken_name_rsc = "ancient trinket"
    StatsResetToken_icon_rsc = srtoken.bgf
    StatsResetToken_desc_rsc = \
-      "This token grants you one stat reset.  Seek out a village elder "
-      "who may offer you a statistics reset in exchange for this item. "
+      "This is a Trinket of the Elders, said to be handed to them by the Gods. "
+      "Legend holds that these Ancient Trinkets contain the power to change "
+	  "the very structure of a Mortal. Surely the Elders would greatly "
+	  "reward anyone who returns this Trinket to them."
```

Comment:
Please change these tabs to spaces... also set up your Notepad++ so all tabs are spaces (Settings->Preferences->Tab Settings, Tab size: 3, check Replace by space).